mne/realtime/fieldtrip_client.py

```diff
@@ -173,6 +173,10 @@ def _guess_measurement_info(self):
                     this_info['kind'] = FIFF.FIFFV_MISC_CH
                 elif ch.startswith('SYS'):
                     this_info['kind'] = FIFF.FIFFV_SYST_CH
+                else:
+                    # unknown channel type, do not include
+                    warn('Unknown channel type in FieldTrip header: %s' % ch)
```

Comment:
A perhaps better way to do this is construct a list of such channel names in the loop, and warn once at the conclusion of the loop if so. Reduces warnings from 4/5 to 1 in the CHPI case, for example.